<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Design</title>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <script src="js/jquery.js"></script>
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <script src="js/bootstrap.min.js"></script>
    <link rel="stylesheet" href="css/general.css">
    <script src="js/general.js"></script>
    <link rel="stylesheet" href="css/article.css">
</head>
<body>
<nav id="nav-bar" class="navbar navbar-expand-lg navbar-light sticky-top"></nav>
<div id="title" class="col-12" style='-o-background-image: url("https://2021.igem.org/wiki/images/b/ba/T--DUT_China--t_design.jpg");
    -moz-background-image: url("https://2021.igem.org/wiki/images/b/ba/T--DUT_China--t_design.jpg");
    -webkit-background-image: url("https://2021.igem.org/wiki/images/b/ba/T--DUT_China--t_design.jpg");
    background-image: url("https://2021.igem.org/wiki/images/b/ba/T--DUT_China--t_design.jpg");'>
    <div>Design</div>
</div>
<section id="main-text">
    <article class="article">
        <h1>Design</h1>
        <p>In 2016, Yoshida et al. [1] reported the discovery of the bacterium, Ideonella
            sakaiensis 201-F6, which develops
            a two-enzyme system to deconstruct PET plastic to TPA and EG molecules, which could be effectively
            catabolized
            as a carbon and energy sources for cells. Further research of this two enzymes system reveals that enzyme
            PETase
            is a cutinase-like serine hydrolase that attacks the PET plastic polymer with the highest efficiency in mild
            temperature among all previously discovered PET degradation enzymes. The metagenome-derived leaf-branch
            compost
            cutinase (LCC) shows higher activity on low crystallinity PET film degradation [2]. However, LCC is still
            not
            generally applicable due to the cost and high applied optimum temperature around 72 °C [2], which highly
            limit
            its further application in engineered bacteria bio-degradation system. Furthermore, the percentage of PET
            degradation intermediates MHET generated via LCC were near to 60% [3], which could not be removed by MHETase
            effectively due to the possible loss of its enzymatic activity at LCC applied temperature. Compared to LCC,
            the
            MHET removing efficiency in PETase and MHETase coexisting system was near to 100% [4]. LCC reveals near 5.5
            times and 4 times lower enzymatic activity to PET film and high crystallinity PET substrate at 30 °C
            compared to
            PETase [1] [5]. PETase was considerably more active against PET film at low temperatures than other PET
            degradation enzymes including TfH, LCC, and FsC [1]. And more efforts of PETase protein rational evolution
            are
            made and these works largely improved enzyme PETase thermostability with a Tm value that was increased by
            8.81
            °C and reacting activity by around 14-fold at 40 °C [5], whose activity is even higher than LCC and
            BhrPETase
            isolated from bacterium HR29 [3][5]. Furthermore, in a latter research, highly synergistic relationship
            between
            PETase and MHETase was discovered in the conversion of amorphous PET film to monomers [4], which effectively
            accelerate PET degradation. However, this two enzymes system may be still limited due to enzymatic loss
            caused
            by protein fusion, inhibition effects and diffusion of intermediates. And methods of rational evolution of
            protein may not be applicable to further improve the overall turnover rate [6]. In detail, the MHET
            molecules
            produced via PET degradation may be competitive to PETase active sites, which was revealed via calculation
            results shown in this work. Despite of this, MHET molecules diffusion in space may also be a problem for
            MHETase
            remove MHET in time.</p>
        <div class="figures">
            <div class="row">
                <img src="https://2021.igem.org/wiki/images/f/f8/T--DUT_China--design11.png" alt=""
                     class="col-12 col-md-8 mx-auto">
            </div>
        </div>
        <p class="caption text-md-center">Figure 1: PET degradation pathway</p>


        <p>Therefore, in this work, we designed an intricate multidomain protein scaffold composed of short peptide tags
            of
            RIAD [8] and RIDD [9], enzymes of PETase and MHETase and protein hydrophobin4 [8], in which the enzymes
            constructed in near positions to each other may work with highly synergistic relationship. All proteins
            involved
            have unique functions in this system. PETase and MHETase are two enzymes involved deconstructing polymer PET
            plastic to MHET and MHET to TPA molecules, respectively. And hydrophobin4, a small fungal protein, possess
            positive effects on altering the physicochemical properties of PET surfaces and enzyme aggregation
            enhancement
            when it was fused with PET degradation enzyme cutinase [10]. Here, hydrophobin4 are involved in our designed
            system with possible functions of adhering to PET polymers and altering the physicochemical properties of
            PET
            for degradation improvement [11]. The peptides of RIDD and RIAD originated from cAMP-dependent protein
            kinase
            (PKA) and the A kinase-anchoring proteins (AKAPs), respectively. The RIAD peptide is capable of specifically
            binds to the RIDD dimer with strong affinity [12]. The following two features make them ideal protein
            binding
            modular for our system assembly: (1) the tiny size (44 and 18 amino acids, respectively), which minimizes
            the
            disturbances to the structure and activity of the enzymes when fused with these peptides, (2) the strong
            binding
            affinity (with a KD of 1.2 nM between RIDD dimer and RIAD peptide demonstrated in our colleagues’ previous
            work
            [12]) to ensure the stability of the whole enzyme complexes in the environment.</p>
        <div class="figures">
            <div class="row">
                <img src="https://2021.igem.org/wiki/images/a/ab/T--DUT_China--design12.png" alt=""
                     class="col-12 col-md-8 mx-auto">
            </div>
        </div>

        <p class="caption">Figure 2: Schematic diagram of this system[12] (The assembly of tri-enzyme units.
            E1,
            E2: enzymes; green and blue structure: RIDD dimer; black line: linker; pink structure: RIAD; one orange
            circle:
            cysteine; two orange circles: disulfide bond)</p>
        <div class="figures">
            <div class="row">
                <img src="https://2021.igem.org/wiki/images/b/bb/T--DUT_China--design13.png" alt=""
                     class="col-12 col-md-8 mx-auto">
            </div>
        </div>
        <p class="caption text-md-center">Figure 3: Three-protein complex assembly schematic diagram.</p>
        <p>In summary, we plan to construct a three-enzyme complex of RIDD-PETase, RIDD-MHETase and RIAD-hydrophobin4
            that
            is combined with RIDD-RIDD-RIAD (Figure 3), and use this complex to degrade PET plastic effectively.</p>
        <div class="figures">
            <div class="row">
                <img src="https://2021.igem.org/wiki/images/5/5a/T--DUT_China--design14.png" alt=""
                     class="col-12">
            </div>
        </div>

        <p class="caption text-md-center">Figure 4: Plasmid construction stagey</p></article>
    <article class="article"><h1>Reference</h1>
        <p>
            [1] Shosuke Yoshida, Kazumi Hiraga, Toshihiko Takehana, Ikuo Taniguchi, Hironao Yamaji, Yasuhito Maeda,
            Kiyotsuna Toyohara, Kenji Miyamoto, Yoshiharu Kimura, Kohei Oda, A bacterium that degrades and assimilates
            poly (ethylene terephthalate), 2016.
        </p>
        <p>
            [2] V. Tournier, C. M. Topham, A. Gilles, B. David, C. Folgoas, E. Moya-Leclair, E. Kamionka, M.-L.
            Desrousseaux, H. Texier, S. Gavalda, M. Cot, E. Guémard, M. Dalibey, J. Nomme, G. Cioci, S. Barbe, M.
            Chateau, I. André1, S. Duquesne, A. Marty, An engineered PET depolymerase to break down and recycle plastic
            bottles, 2020.
        </p>
        <p>
            [3] Xingxiang Xi, Kefeng Ni, Helong Hao, Yuepeng Shang, Bo Zhao, Zhen Qian, Secretory expression in Bacillus
            subtilis and biochemical characterization of a highly thermostable polyethylene terephthalate hydrolase from
            bacterium HR29, 2021.
        </p>
        <p>
            [4] Brandon C. Knotta, Erika Ericksona, Mark D. Allenb, Japheth E. Gado, Rosie Graham, Fiona L. Kearns,
            Isabel Pardo, Ece Topuzlu, Jared J. Anderson, Harry P. Austin, Graham Dominick, Christopher W. Johnson,
            Nicholas A. Rorrer, Caralyn J. Szostkiewicz, Valérie Copié, Christina M. Paynec, H. Lee Woodcock, Bryon S.
            Donohoef, Gregg T. Beckham, and John E. McGeehan, Characterization and engineering of a two-enzyme system
            for plastics depolymerization, 2019.
        </p>
        <p>
            [5] Hyeoncheol Francis Son, In Jin Cho, Seongjoon Joo, Hogyun Seo, Hye-Young Sagong, So Young Choi, Sang Yup
            Lee, Kyung-Jin Kim, Rational Protein Engineering of Thermo-Stable PETase from Ideonella sakaiensis for
            Highly Efficient PET Degradation, 2019.
        </p>
        <p>
            [6] Erickson E, Shakespeare TJ, Bratti F, Buss BL, Graham R, Hawkins MA, König G, Michener WE, Miscall J,
            Ramirez KJ, Rorrer NA, Zahn M, Pickford AR, McGeehan JE, Beckham G, Comparative performance of PETase as a
            function of reaction conditions, substrate properties, and product accumulation, 2021.
        </p>
        <p>[7] Boneta S, Arafet K, Moliner V, QM/MM Study of the Enzymatic Biodegradation Mechanism of Polyethylene
            Terephthalate, 2021.
        </p>
        <p>
            [8] Carlson, C. R. et al. Delineation of type I protein kinase A-selective signaling events using an RI
            anchoring disruptor. J. Biol. Chem. 281, 21535–21545, 2006. 24.
        </p>
        <p>[9] Gold, M. G. et al. Molecular basis of AKAP specificity for PKA regulatory subunits. Mol. Cell 24,
            383–395, 2006.
        </p>
    </article>
</section>
<div id="footer-point"></div>
<script>
    //About the top-scroll--linked to general.js
    window.onload = init;
</script>
<script>
    //Include the navigation bar, the top-scroll and the footer
    $(document).ready(function () {
        $('#nav-bar').load('top.html');
        $('#footer-point').load('footer.html');
    });
</script>
</body>
</html>